server/sandbox_stop.go

```diff
@@ -24,10 +25,13 @@ func (s *Server) StopPodSandbox(ctx context.Context, req *pb.StopPodSandboxReque
 	if err != nil {
 		return nil, err
 	}
-
-	if err := s.netPlugin.TearDownPod(netnsPath, podNamespace, sb.id, podInfraContainer.Name()); err != nil {
-		return nil, fmt.Errorf("failed to destroy network for container %s in sandbox %s: %v",
-			podInfraContainer.Name(), sb.id, err)
+	if _, err := os.Stat(netnsPath); err == nil {
```

Comment:
you need an additional check for the teardown then...

```
if err :=stat(netnsPath) - err != nil && !os.IsNotExists(err) {
return err
}
if err == nil { // because it can be os.NotExist
  teardown
}
```